supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAttributes.scala

```diff
@@ -90,4 +92,16 @@ object JesAttributes {
       case _ => "genomics.default-zones was set but no values were provided".invalidNel
     }
   }
+
+  def validateQps(config: Config): ErrorOr[Int Refined Positive] = {
+    import eu.timepit.refined._
+
+    val qp100s = config.as[Option[Int]]("genomics-api-queries-per-100-seconds").getOrElse(GenomicsApiDefaultQps)
+    val qpsCandidate = qp100s / 100
+
+    refineV[Positive](qpsCandidate) match {
+      case Left(_) => s"Supplied QPS for Google Genomics API was not a positive integer (supplied value was $qpsCandidate".invalidNel
```

Comment:
`)` missing at the end. Also the actual supplied value was `qp100s` not `qpsCandidate` no ? It might be confusing if someone get this error message, look at their conf and see a different number.